trade-finance-manager-api/src/v1/routes.js

```diff
@@ -85,6 +86,10 @@ openRouter.route('/deals/submit').put(dealSubmit.submitDealPUT);
 
 authRouter.route('/deals/submitDealAfterUkefIds').put(dealSubmit.submitDealAfterUkefIdsPUT);
 
+authRouter.route('/deals').get(dealController.getDeals);
+
+authRouter.route('/deals/:dealId').get(dealController.getDeal);
```

Comment:
should you add validation for deal ID here?
`....get(validation.dealIdValidation, handleValidationResult, dealController.getDeal);`

Reply:
Great catch! Definitely should thanks for spotting